### ISSUE TITLE
Is nucleus dead/abandoned?

### ISSUE THREAD

**Issue by rex** [holidaypirates/nucleus#68]
It's to the point now with Node `8.9` that nucleus just straight up does not work, throws exceptions every time it runs. A cursory look through the issues indicates near or total abandonment. It's fine, I can imagine how difficult and time consuming it is to maintain a library that lots of people like and use. 

Unless this is still being maintained my team and I are going to have to find another styleguide generation solution 🙁

**Comment by pkkummermo** [comment]
I hope not, but the project has been quite as of late. Perhaps @michaelseibt can shed some light? There are PRs waiting which would be awesome to merge.

**Comment by michaelseibt** [comment]
Hey everybody. All my apologies for the huge queue of issues and requests piling up. One reason is undoubtedly the lack of time to handle both: Maintaining the current version and breaking our heads about the next one.

As written here [1], taking a look at the feature request makes it pretty clear that Nucleus is built upon a long list of assumptions, for example using SASS or forcing the atomic paradigm (instead of giving a choice), and furthermore is not able to be extended with custom annotations that easily.

I'll go through the list of issues and PRs and evaluate which issues should be fixed immediately, as well as pushing an update and getting it to run on current node versions.

Thanks for all your patience! This project won't be abandoned without a clear notice.

[1] https://github.com/holidaypirates/nucleus/issues/43#issuecomment-278724358

**Comment by evanwills** [comment]
@michaelseibt Thanks for the update.

**Comment by michaelseibt** [comment]
Hey all, a very late and quick update on the status of this project for the sake of transparency. I handed over the project to HolidayPirates, a company I had been working with until 2018 (check out their travel blog!). Back then, we've not had plans for an agreement about keeping this project maintained by external contributors. At this point, it's pretty clear that the same applies for internal contributors.

To sum it up, this one here has gone down to Davy Jones’ Locker. @holidaypirates can you mark this repository as deprecated if there are no other plans about it?
